### PR TITLE
Remove overviews

### LINKED ISSUE #136: In the overview document on how to implement IPFS - include links to specs
In the [Implement IPFS](https://github.com/ipfs/specs/blob/master/overviews/implement-ipfs.md) overview document, there are links to reference implementations, but no links to the specifications.

### PR DESCRIPTION
**Pull request by hsanjuan**
These documents are oudated, probably misleading and the specs is no place
for non-spec documentation (or at least non-updated spec documentation).

Fixes #136.

### CODE REVIEW

**Comment by hsanjuan** [comment]
/remind me to merge this in two weeks if no-one blocks.

**Comment by hsanjuan** [comment]
/remind me to merge this in two weeks.

File: overviews/implement-api-bindings.md [hunk 1]
```diff
@@ -1,171 +0,0 @@
-# IPFS API Implementation Doc
```
Comment (Stebalien) [inline overviews/implement-api-bindings.md]: This looks like it may still be useful & accurate.

File: overviews/implement-ipfs.md [hunk 2]
```diff
@@ -1,200 +0,0 @@
-# IPFS Implementation Doc
```
Comment (Stebalien) [inline overviews/implement-ipfs.md]: :+1: this is out of date enough that it's not particularly useful.

File: overviews/implement-api-bindings.md (hunk 1 above)
Comment (hsanjuan) [reply overviews/implement-api-bindings.md]: @johnnymatthews we'd need to recycle this into the docs. Maybe along with the API reference?

Comment (hsanjuan) [reply overviews/implement-api-bindings.md]: I opened https://github.com/ipfs/docs/issues/478

**Comment by hsanjuan** [comment]
Merged making sure we track useful information for the docs site.

File: overviews/implement-api-bindings.md (hunk 1 above)
Comment (hsanjuan) [reply overviews/implement-api-bindings.md]: It seems the docs are duplicated here https://github.com/ipfs/go-ipfs/blob/master/docs/implement-api-bindings.md

**Pull request merged**